### PR TITLE
UI 2.0 Sidebar 1687 and fix: 1823 Log Out button 

### LINKED ISSUE #1687: [UI 2.0] SideBar
**What would you like to be added**:
The new sidebar for UI 2.0. Please check:  https://github.com/Seneca-CDOT/telescope/issues/1676

Let's use [MUI Icons](https://material-ui.com/components/material-icons/)

| Icon 	| Icon on MUI icons: 	|
|------	|--------------------	|
|     search 	|         `import SearchIcon from '@material-ui/icons/Search';`           	|
|   home   	|            `import HomeIcon from '@material-ui/icons/Home';`        	|
|    about us  	|          `import InfoIcon from '@material-ui/icons/Info';`          	|
|     log in 	|           `import AccountCircleIcon from '@material-ui/icons/AccountCircle';`         	|
|    dark/light theme  	|    `import InvertColorsIcon from '@material-ui/icons/InvertColors';`                	|

### PR DESCRIPTION
**Pull request by DukeManh**
## Issue This PR Addresses

Fixes #1687 and #1823

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/src/components/Header/DesktopHeader.tsx [hunk 1]
```diff
@@ -19,46 +22,50 @@ import Login from '../Login';
 
 const useStyles = makeStyles((theme) => ({
   root: {
+    width: '15em',
+    height: '32em',
+    flexDirection: 'column',
+    justifyContent: 'end',
+    position: 'fixed',
+    top: '0',
```
Comment (PedroFonsecaDEV) [inline src/web/src/components/Header/DesktopHeader.tsx:30]: `top: '1.4em'`, 

To achieve this result:
Screen Shot 2021-02-24 at 1 24 13 AM

**Review by PedroFonsecaDEV — CHANGES_REQUESTED** [review]

File: src/web/src/styles/globals.css [hunk 2]
```diff
@@ -16,3 +16,15 @@ body {
...
+  body {
+    margin-bottom: 6.4rem;
+  }
+}
+
+@media only screen and (max-width: 480px) {
+  body {
+    margin-bottom: 5.6rem;
+  }
+}
```
Comment (PedroFonsecaDEV) [inline src/web/src/styles/globals.css:30]: Move this to a component level instead of a global level.

**Comment by DukeManh** [comment]
@PedroFonsecaDEV I removed the global body padding. I managed to add a hide animation to the navbar so that it doesn't cover anything.

**Review by PedroFonsecaDEV — DISMISSED** [review]
Excellent work!

File: src/web/src/components/Header/MobileHeader.tsx [hunk 3]
```diff
@@ -1,151 +1,90 @@
...
-import Logo from '../Logo';
 import Login from '../Login';
-// 1.7 band-aid (Removing theme | line 11)
-// import ThemeToggleButton from '../ThemeToggleButton';
+
+/**  This will solve the problem of incorrect rendering of theme icon when theme preference is dark
+ * This ensures that the version displayed to user is the client view which ties to the client's preference theme.
+ * This is only an issue on DesktopHeader since on MobileHeader there is a listener triggering rerendering.
+ * */
+// 1.7 band-aid (Removing theme | line 16 - 18)
```
Comment (Metropass) [inline src/web/src/components/Header/MobileHeader.tsx:17]: Just clean up the commented out code, and it's good! Nice work! 😄

**Review by Metropass — CHANGES_REQUESTED** [review]

File: src/web/src/components/Header/MobileHeader.tsx (hunk 3 above)
Comment (DukeManh) [reply src/web/src/components/Header/MobileHeader.tsx:17]: @tonyvugithub wrote that comment, maybe he wants in the future? I'll let him decide on this.

Comment (PedroFonsecaDEV) [reply src/web/src/components/Header/MobileHeader.tsx:17]: @Metropass @DukeManh This comment is mine from our 1.7 release.

Comment (Metropass) [reply src/web/src/components/Header/MobileHeader.tsx:17]: @PedroFonsecaDEV Noted! Approved

**Review by Metropass — APPROVED** [review]

File: src/web/src/components/Header/MobileHeader.tsx (hunk 3 above)
Comment (DukeManh) [reply src/web/src/components/Header/MobileHeader.tsx:17]: Do you want to keep it?

Comment (PedroFonsecaDEV) [reply src/web/src/components/Header/MobileHeader.tsx:17]: My comment is only the line `// 1.7 band-aid (Removing theme | line 16 - 18)` 
Let's keep it.

**Review by PedroFonsecaDEV — APPROVED** [review]

**Pull request merged**